assets/_scss/elements/_list.scss

```diff
@@ -21,7 +21,7 @@ article {
     }
```

Comment:
If you make unstyled list the default and styled-list what content creators need to use to create what is otherwise an html default expected behavior it will cause greater confusion.